src/main/webapp/index.html

```diff
@@ -32,10 +32,11 @@ <h2>How it works</h2>
         <h2>Translated Sentence Analyzer</h2>
         <p>Type/Paste your text below:</p>
        <form method="POST" action="/">
-        <textarea name="user-message"></textarea>
+        <textarea name="user-message" id="message"></textarea>
```

Comment:
Maybe display what the user's message was also?

Reply:
I agree! Showing what the user originally input would be nice!

Reply:
I'll add in that change to the JS and update ID name